### PR TITLE
Added some scripts to emulate the image via qemu

### LINKED ISSUE #393: To add a way to create automated Raspbian images via QEMU and Travis
**Is your feature request related to a problem? Please describe.**
To fix the problem of manually creating images with win32imager

**Describe the solution you'd like**
An approach suggested by @xeon-zolt during jugaadfest to use qemu with travis and release a new image on every PR.

**Additional context**
42953135_299959934153269_4223152633902268416_n

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #393 

* Contents of This PR *
- I have created a barebones custom qcow image that contains the `travis_install.sh` in `/home/pi` directory
- The travis script is booted when the qcow image is emulated via QEMU using a Jessie kernel(as it was the only working kernel I could find) {please suggest another one if find another one}, the script is configured via the rc.local file.
- And the rest is self-explanatory

### CODE REVIEW

**Comment by hongquan** [comment]
Our Raspbian is based on Debian Stretch, why do you use Jessie kernel in QEMU?

**Comment by hongquan** [comment]
@stealthanthrax Please provide the log of your script.

**Comment by stealthanthrax** [comment]
@hongquan , I have completed this PR , please review.

What kind of logs are you refereing to?
> @stealthanthrax Please provide the log of your script.

**Comment by hongquan** [comment]
@stealthanthrax the log produced when your script run. It is the first log that people can think.

**Comment by hongquan** [comment]
Using this vague URL for Raspbian Qcow image https://drive.google.com/uc?export=download&confirm=DBZl&id=1CJj3sefBr5cj3k-88WSAXbtITg1EU_hW

is not good for long-term maintenance, because developer cannot look into this URL to know if the Raspbian base image is up-to-date or not. At that time, if Raspbian latest release is, 2019-03-01, for example, the developer/maintainer cannot know if your Raspbian Qcow is updated or not.

Note: This is just a warning for future improvement. We can accept this GDrive solution now, but should think about replacement solution later.

File: .travis_scripts/travis_script.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Downloading and installing QEMU from the source
+wget https://download.qemu.org/qemu-2.12.0-rc0.tar.xz
+tar xvJf qemu-2.12.0-rc0.tar.xz
+cd qemu-2.12.0-rc0
+./configure --target-list=arm-softmmu,aarch64-softmmu
+make
+
+# Downloading the latest Raspbian Qcow
+wget https://drive.google.com/uc?export=download&confirm=DBZl&id=1CJj3sefBr5cj3k-88WSAXbtITg1EU_hW
```
[the remark this answers is not in the document]
Comment (xeon-zolt) [reply .travis_scripts/travis_script.sh]: better to keep the file in the repo then downloading from 3rd party source

[the remark this answers is not in the document]
Comment (hongquan) [reply .travis_scripts/travis_script.sh]: @xeon-zolt No. Binary file should not be hosted in Git, except when it is "part of our source". This qwow2 file is converted from https://www.raspberrypi.org/downloads/raspbian/ and truly a 3rd-party file. Also it is 1.7GB. Don't abuse Git to host that file (Don't force other developers to clone that 1.7GB).

**Comment by stealthanthrax** [comment]
@hongquan , updated

**Comment by stealthanthrax** [comment]
I guess, not relevant anymore.

**Pull request closed without merging**